--- setup.py.orig	2009-03-26 12:00:15.605813505 +0100
+++ setup.py	2009-03-26 12:00:20.109564509 +0100
@@ -141,21 +141,6 @@
             except OSError:
                 log.warn("Could not call %s.", cmd)
         
-        log.info("updating MIME types")
-        try_to_call(["update-mime-database", mime_dir])
-        
-        log.info("updating desktop database")
-        try_to_call(["update-desktop-database"])
-        
-        for loc in ["gnome", "hicolor"]:
-            icon_dir = join(self.prefix, "share", "icons", loc)
-            
-            log.info("updating icon cache %s", icon_dir)
-            try_to_call(["gtk-update-icon-cache", "-q", "-f", "-t", icon_dir])
-        
-        log.info("updating scrollkeeper database")
-        try_to_call(["scrollkeeper-update", "-q", "-o", omf_dir])
-    
     def remove(self, path):
         """
         Wraps the remove function, so that the dry_run parameter doesn't need
